README.md

```diff
@@ -12,11 +12,11 @@ rav1e is an AV1 video encoder. It is designed to eventually cover all use cases,
 * 64x64 superblocks
 * 4x4 to 64x64 RDO-selected square and 2:1/1:2 rectangular blocks
 * DC, H, V, Paeth, smooth, and a subset of directional prediction modes
-* DCT, ADST and identity transforms (up to 64x64, 16x16 and 32x32 respectively)
+* DCT, ADST, and identity transform (up to 64x64, 16x16 and 32x32 respectively)
 * 8-, 10- and 12-bit depth color
 * 4:2:0 (full support), 4:2:2 and 4:4:4 (limited) chroma sampling
 * Variable speed settings
-* Near real-time encoding at high speed levels
+* Near real-time encoding at high-speed levels
```

Comment:
This was more correct before.  Rather than referring to a single "high-speed" level (like a "high-speed train"), it is referring to high speed levels in general (the speed levels are high, rather than a level being high-speed).